apps/E2E/wdio.conf.windows.js

```diff
@@ -122,7 +122,7 @@ exports.config = {
    * @param {Array.<Object>} capabilities list of capabilities details
    * @param {Array.<String>} specs List of spec file paths that are to be run
    */
-  before: async function () {
+  before: async () => {
```

Comment:
Should this (as well as the other cases/comments in the other wdio.conf files) be changed too?

Reply:
Updated all WDIO files with this pattern. I don't think the commented ones matter atm